### ISSUE TITLE
Suggestion

### ISSUE THREAD

**Issue by Yoon0117** [thejoabo/autofishbot#120]
Works perfectly, much better than I imagined it would!

I think you should add a feature to use the Personal booster (I have 250 right now and can't use with this bot)
Also option to type custom text, just in case the bot doesn't have all the features, and if someone talks at the user, it would seem like a bot if there's no response but non-stop fishing

**Comment by thejoabo** [comment]
@Yoon0117, awesome to hear that it's working for you ! I'm actually thinking about revamping the entire project with TypeScript and Electron to give it a better GUI and provide a more robust package manager to help users.  About that Personal booster feature and custom text option, those sound like cool ideas—I'll definitely consider them! Thanks for the suggestions! (I'm still not sure how would I detect context to infer that someone talked to the user to stop fishing and respond, that's why I recommend to always fish on private channels)

**Comment by Yoon0117** [comment]
Yeah, I think that's the best option too. Thanks for the quick response!
